ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -66,7 +66,7 @@ export default function OnboardingFlow() {
       return;
     }
 
-    if (isInitialized && !isUnlocked) {
+    if (!isInitialized && isUnlocked) {
```

Comment:
Was this incorrect? This seems like an important change if so. Maybe we ought to extract this to a separate PR for visibility purposes and write a test to catch this in the future?

Reply:
@mcmire I made this change because @adonesky1 write one bullet task for this. Here is description of it:

`After creating a password, it shows the login page briefly with a loading overlay before navigation to the success screen. This shouldn't happen. It should lead directly to the success screen.`

When we have `isInitialized && !isUnlocked` it shows the login page briefly with loading overlay before navigation to the success screen. I will share you screencast what I made when we have this condition in `onboarding-flow.js` file.

https://user-images.githubusercontent.com/92527393/156780038-94855e2f-48d0-4a5e-91f4-6d54c98cd24e.mov

Maybe is good to extract this to separate PR. What do you think about this @adonesky1?

Reply:
@mcmire @adonesky1 I didn't revert changes in `onboarding-flow.js` file. I will wait for your comments about this.

Reply:
I'm messing around with this and I agree with @mcmire that we should extract this into a separate PR, since its a bit tricky and requires a bit more direct thought.

Reply:
Good call out @mcmire

Reply:
Thank you @adonesky1 for respond on this. I revert changes in `onboarding-flow.js` file here: ea104e72152e04834f36cd41dac5754ad20b2190